src/Signer/Ecdsa/KeyParser.php

```diff
@@ -1,93 +0,0 @@
-<?php
-declare(strict_types=1);
-
-namespace Lcobucci\JWT\Signer\Ecdsa;
-
-use InvalidArgumentException;
-use Lcobucci\JWT\Signer\Key;
-use Mdanter\Ecc\Crypto\Key\PrivateKeyInterface;
-use Mdanter\Ecc\Crypto\Key\PublicKeyInterface;
-use Mdanter\Ecc\Math\GmpMathInterface;
-use Mdanter\Ecc\Serializer\PrivateKey\DerPrivateKeySerializer;
-use Mdanter\Ecc\Serializer\PrivateKey\PemPrivateKeySerializer;
-use Mdanter\Ecc\Serializer\PrivateKey\PrivateKeySerializerInterface;
-use Mdanter\Ecc\Serializer\PublicKey\DerPublicKeySerializer;
-use Mdanter\Ecc\Serializer\PublicKey\PemPublicKeySerializer;
-use Mdanter\Ecc\Serializer\PublicKey\PublicKeySerializerInterface;
-use const PHP_EOL;
-use function count;
-use function preg_match;
-use function str_replace;
-
-class KeyParser
```

Comment:
This BC break requires documentation

Reply:
This class was always considered as internal, but we were not using `@internal` yet. I'm not sure if this really need to be stated as BC-break :(